blog/2024/04-28-jellyfin-release-10.9.0/index.mdx

```diff
@@ -0,0 +1,124 @@
+
+* Our minimum FFmpeg version has been bumped from 4.0 to 4.4 to take advantage of many new features. For users of our Jellyfin FFmpeg build (Docker, `.deb`/Debuntu repo, and Windows installs), you're already on an even newer version now with FFmpeg 6.0.1, but this minimum change would affect any other platforms.
+
+* The server now supports in-process restarting, and removes the old hacky `restart.sh` method. This should make things like installing plugins much more robust.
+
+* Uploaded images are now subject to much more verification, helping plug several security holes in this feature.
+
+* The backend SQLite database now supports connection pooling, which should improve performance for database operations.
+
+* Support for the HappyEyeballs engine improving dual-stack IPv4/IPV6 network support.
+
+* Improved handling of IP binding to fully respect these settings (including DLNA and SSDP).
+
+### Packaging
+
+* The `jellyfin` service user is now automatically added to the `render` and `video` groups on Debian/Ubuntu systems for new installs, helping improve the onboarding for hardware acceleration.
+
+* Packaging is now unified with a new CI system, as outlined in [our previous blog post](https://jellyfin.org/posts/new-ci-new-repo). As a normal user you should see no change here, but this helps massively improve the build process, and if you are building your own packages please reference [the new packaging repository](https://github.com/jellyfin/jellyfin-packaging) for the updated process.
+
+### Transcoding & FFmpeg
+
+* Several audio improvements: support for Opus, FLAC, and ALAC codecs in the HLS engine; improved support and handling for transcoding DTS and TrueHD audio streams; new codec enforcement; better bitrate calculations; new audio normalization features; and stereo downmixing algorithm selection.
+
+* FFmpeg segments are now automatically deleted during transcoding, reducing the space of the transcoding temporary directory.
+
+* Support for direct playback of DVD and BluRay data folders, preferred over ISOs.
+
+* Support for AV1 hardware and software encoding.
+
+* Support for RK3588 hardware acceleration via MPP.
+
+* Support for Vulkan-based tonemapping with AMD VA-API.
+
+* Support for hardware acceleration on Apple systems (both Apple Silicon and Intel).
+
+* Support for Dolby Vision playback remuxing.
+
+### Scanning, Library & Playlist Management
+
+* Support for lyric scanning and management.
+
+* Season names are now parsed from NFO files.
+
+* The MusicBrainz interface has been improved to better handle music libraries.
+
+* Support for hearing-impaired subtitles (sdh/hi./cc).
+
+* Support for playlist ACLs with playlist sharing between users and enforcement of playlist ownership.
```

Comment:
You might want to add a comment here, that existing playlists will be creator-only and permissions for other users need to be manually assigned once if they should be share/global. Playlists sourced from files in a library are always global.

Reply:
Added additional details